### PR TITLE
dbeaver/pro#1584 hide debug button from TE managers

### PR DESCRIPTION
**Pull request by LonwoLonwo**
2023-05-08 13_12_28-Active Session Information

I didn't find anything else for Managers. Only debug button.
If you find disabled for managers buttons - please message me.

### CODE REVIEW

**Review by E1izabeth — APPROVED** [review]

**Review by ShadelessFox — APPROVED** [review]

**Comment by uslss** [comment]
Create connection and Cloud explorer buttons are available for now
image

**Comment by LonwoLonwo** [comment]
fixed

2023-05-09 14_46_24-DBeaver Team Edition 23 0 4 - actor

File: plugins/org.jkiss.dbeaver.core/plugin.xml [hunk 1]
```diff
@@ -519,10 +519,10 @@
 
                 <separator name="driverGroup" visible="true"/>
                 <command commandId="org.jkiss.dbeaver.core.new.connection">
-                    <visibleWhen checkEnabled="true"/>
+                    <visibleWhen checkEnabled="true"><test property="org.jkiss.dbeaver.core.global.canCreateConnection"/></visibleWhen>
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.core/plugin.xml:522]:
```suggestion
                    <visibleWhen checkEnabled="true">
                        <test property="org.jkiss.dbeaver.core.global.canCreateConnection"/>
                    </visibleWhen>
```

File: plugins/org.jkiss.dbeaver.core/plugin.xml [hunk 2]
```diff
@@ -519,10 +519,10 @@
...
                 </command>
                 <command commandId="org.jkiss.dbeaver.core.new.connection.from.url">
-                    <visibleWhen checkEnabled="true"/>
+                    <visibleWhen checkEnabled="true"><test property="org.jkiss.dbeaver.core.global.canCreateConnection"/></visibleWhen>
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.core/plugin.xml:525]:
```suggestion
                    <visibleWhen checkEnabled="true">
                        <test property="org.jkiss.dbeaver.core.global.canCreateConnection"/>
                    </visibleWhen>
```

File: plugins/org.jkiss.dbeaver.core/plugin.xml (hunk 1 above)
Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.core/plugin.xml:522]: I do not see any specific reasons for that.

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.core/plugin.xml:522]: Readability.

**Pull request merged**